types/src/on_chain_config/aptos_features.rs

```diff
@@ -104,6 +104,17 @@ pub enum FeatureFlag {
     /// that results in a new package created but without any code. With this feature, it is no
     /// longer possible and an explicit error is returned if publishing is attempted.
     DISALLOW_INIT_MODULE_TO_PUBLISH_MODULES = 82,
+    /// We keep the Call Tree cache and instruction (per-instruction)
+    /// cache together here.  Generally, we could allow Call Tree
+    /// cache and disallow instruction cache, however there's little
+    /// benefit of such approach: First, instruction cache requires
+    /// call-tree cache to be enabled, and provides relatively little
+    /// overhead in terms of memory footprint. On the other side,
+    /// providing separate choices could lead to code bloat, as the
+    /// dynamic config is converted into multiple different
+    /// implementations. If required in the future, we can add a flag
+    /// to explicitly disable the instruction cache.
+    ENABLE_CALL_TREE_AND_INSTRUCTION_VM_CACHE = 83,
```

Comment:
Do you want to add it as default flag? This will get it to devnet for 1.26 in this case + all aptos tests will use new caches - should be ok because gas costs are less?